content/rs/concepts/compatibility.md

```diff
@@ -81,7 +69,6 @@ follows:
     - zset-max-ziplist-entries
     - zset-max-ziplist-value
 
-## Compatibility with Open Source Redis cluster
+## Compatibility with open source Redis Cluster API
 
-RS supports [Redis OSS cluster
-protocol]({{< relref "/rs/concepts/data-access/oss-cluster-api.md" >}}) if it is enabled for a database.
+Redis Enterprise Software supports [Redis OSS Cluster API]({{< relref "/rs/concepts/data-access/oss-cluster-api.md" >}}) if it is enabled for a database. For more information, see [Using the OSS Cluster API]({{< relref "/rs/administering/designing-production/networking/using-oss-cluster-api.md" >}})
```

Comment:
AFAICT, "OSS" hasn't been defined in this article.  Should it be?

Reply:
Added OSS definition to first use of "open source Redis"